nbs/Environments/13_EnvTwoStatesTwoActions.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<p>This should be called risk-reward dilemma. Two States two actions is too generic. Please make sure also to update the description to better motive the environment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BarfussLab/pyCRLD/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>